contributing.md

```diff
@@ -122,15 +122,24 @@ Dependencies are automatically updated by [Dependabot](https://dependabot.com/)
 directly into GitHub.  Each week Dependabot will create a large number of individual PRs that update
 each dependency.  To merge those into main, use the following process:
 
-
 ### Merge Dependabot PRs
-1. Dependabot will create many individual PRs against the `main` branch, which has no
-branch protections and so each of those PRs can be merged automatically by approving each PR with the comment:
+
+#### Manually
+1. Dependabot will create many individual PRs against the `main` branch, which must pass all CI checks and be approved before merging.  Approve each PR with the following comment and if they pass CI they will merge automatically:
 
         @dependabot squash and merge
 
 All done!
 
+#### Automatically
+1. Install and configure the [Github CLI](https://github.com/cli/cli)
+1. From your local Experimenter repo run
+
+        make dependabot_approve
```

Comment:
Nit: excessive rendered indentation

Reply:
This always confuses me.  It says 4 spaces in the doc but then it doesn't render properly?

Reply:
test
    four spaces
        eight spaces

Reply:
It's 4 spaces for a code block (if not using fences).
... But if you're in a list, it might need another 2-4 spaces indentation if you want the code block to be part of the bulleted list.

But yeah, code fences are always better/clearer (and support syntax highlighting, which is a nice bonus).

Reply:
Right exactly